### PR TITLE
Filipino (Tagalog)

### PR DESCRIPTION
**Pull request by HAJDog247**
Not finished

### CODE REVIEW

**Comment by ZkyLB** [comment]
image
i dont think you are supposed to modify this..

**Comment by HAJDog247** [comment]
Oh... If that's the case, I'll take action immediately. I'm so sorry for
that; I didn't know that I shouldn't supposed to change that.

Nonetheless, thank you for letting me know!

File: core/assets/bundles/bundle_tl.properties [hunk 1]
```diff
@@ -88,18 +88,18 @@ committingchanges = Comitting Changes
...
 mods.report = Report Bug
 mods.openfolder = Open Mod Folder
-mod.enabled = [lightgray]Enabled
-mod.disabled = [scarlet]Disabled
+mod.enabled = [lightgray]Nai-bukas
+mod.disabled = [scarlet]Nai-sara
 mod.disable = Disable
-mod.delete.error = Unable to delete mod. File may be in use.
-mod.requiresversion = [scarlet]Requires game version: [accent]{0}
+mod.delete.error = Hindu ma-delete ang mod. Ang file ay baka ginagamit pa.
```
Comment (ZkyLB) [inline core/assets/bundles/bundle_tl.properties]: i was thinking of "Hindi ma-tangal ang mod."
my main language is Filipino/Tagalog, but i suck at it xD

**Comment by HAJDog247** [comment]
WoW! Good to know about that. Well your grammar is a little bit off, but I
might consider it subject to change. "Hindi ma-tanggal ang mod." is the
correct one. The only change is the word "ma-tanggal," since you only typed
in one 'g' instead of two G's.

**Comment by Anuken** [comment]
Stale PR; re-open if needed.

**Pull request closed without merging**